src/vs/workbench/parts/debug/browser/debugStatus.ts

```diff
@@ -24,12 +25,14 @@ export class DebugStatus extends Themable implements IStatusbarItem {
 	private label: HTMLElement;
 	private icon: HTMLElement;
 	private showInStatusBar: string;
+	private viewletId: string;
```

Comment:
I believe `this.viewletId` is not needed

Reply:
It's required for handling configuration change correctly - we need to know if we are in Debug Viewlet when someone changed configuration to hide the status bar item if necessary.